README.md

```diff
@@ -40,12 +40,48 @@ gulp.src('/mysite/**/*', {base: '/mysite'})
   ...
 ```
 
-### Options:
+### Options
 
 - `filerev`: [minimatch](https://github.com/isaacs/minimatch) pattern to filter the files to filerev.
 - `replace`: [minimatch](https://github.com/isaacs/minimatch) pattern to filter the files to replace.
 - `base`: Directory from where the files are served by the web server. Optional, defaults to the [file's base](https://github.com/gulpjs/gulp/blob/master/docs/API.md#optionsbase);
 
+Manifest
+====================
+
+A second plugin is also available that adds a new file to the stream. This file contains a JSON list of all files that were filereved.
+
+## Usage
+
+```javascript
+var filerevReplace = require('gulp-filerev-replace');
+
+// Include all files to filerev and/or replace, making sure the base is correct
+gulp.src('/mysite/**/*', {base: '/mysite'})
+  .pipe(filerevReplace({
+    filerev: ['assets/**/*'], // Select the files to filerev
+    replace: ['views/**/*'],  // Select the files to replace
+    base:    'assets'         // Filerevved files are served from the assets directory by the web server
+  }))
+  .pipe(filerevReplace.addManifest({path: 'assets/filerev.json'}))
+  ...
+```
+
+The example above would add a new file in the stream, with path `assets/filerev.json`, containing something like:
+
+```javascript
+{
+  "assets/loader.gif": "assets/loader-11111111.gif",
+  "assets/styles.css": "assets/styles-22222222.css"
+}
+```
+
+### Options
+
+`options` is an object containing the [vinyl file options](https://github.com/wearefractal/vinyl#file) to use for the manifest file. It defaults to:
+
+- `path`: `filerev_replace_manifest.json`
```

Comment:
One thing I'd nitpick is the `_` versus `-`. The module uses dashes, the example filenames above use dashes. Just that minor sort of thing that screws with your head.